src/drivers/px4fmu/fmu.cpp

```diff
@@ -928,13 +922,23 @@ PX4FMU::update_pwm_rev_mask()
 
 	for (unsigned i = 0; i < _max_actuators; i++) {
 		char pname[16];
-		int32_t ival;
+
+		if (_class_instance == CLASS_DEVICE_PRIMARY) {
```

Comment:
Move this out of the loop, and use `const char* pname_format = "PWM_MAIN_REV%d"`?

Reply:
Done